### PR TITLE
Add fluentd to logging-agent DaemonSet

### PR DESCRIPTION
**Pull request by christianberg**
Add a fluentd container to the logging-agent DaemonSet, with fluentd
config stored in a ConfigMap.

### CODE REVIEW

**Comment by christianberg** [comment]
Work in progress, do not merge.

**Comment by mikkeloscar** [comment]
~@christianberg can you please push to a branch in this repository instead of a fork? This will allow us to run e2e tests once this is ready (we can't run e2e on a fork right now). I gave you write permission to the repo. :)~

Actually, since this is using the logging-agent daemonset which we don't e2e test right now anyway, it probably doesn't matter so much.. :)

**Comment by christianberg** [comment]
Of course! I'll open a new PR and close this one.

**Comment by aermakov-zalando** [comment]
LGTM. Would you mind bumping the reserved CPU/memory accordingly [here](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/config-defaults.yaml#L6)? We sadly don't have this automated yet. No need to do it conditionally, it's fine to just bump it for all clusters.

File: cluster/manifests/logging-agent/daemonset.yaml [hunk 1]
```diff
@@ -105,14 +108,18 @@ spec:
...
+{{ if eq .ConfigItems.logging_fluentd_enabled "true" }}
+        - name: WATCHER_SYMLINK_DIR
+          value: /mnt/fluentd-logs
+{{ end }}
 
         resources:
           limits:
-            cpu: 50m
-            memory: 250Mi
+            cpu: "{{ index .ConfigItems "logging_watcher_cpu" }}"
```
Comment (aermakov-zalando) [inline cluster/manifests/logging-agent/daemonset.yaml:118]: Please just use `.ConfigItems.logging_watcher_cpu`.

**Comment by aermakov-zalando** [comment]
Closed in favor of #1826.

**Pull request closed without merging**